src/task2/task2_1.js

```diff
@@ -0,0 +1,16 @@
+import express from 'express';
+
+import userRouter from './routers/userRouter';
+import { SERVER_PORT } from './config/global';
+import bodyParser from 'body-parser';
```

Comment:
Consider grouping the imports by like "global then projects then local imports".
It will improve readability.